README.rst

```diff
@@ -10,13 +10,17 @@ the `H3 Documentation <https://uber.github.io/h3/>`__.
 Installing
 ==========
 
-You need to have a ``cc`` in your ``$PATH`` when installing this
+You need to have ``cc``, ``make``, and ``cmake`` in your ``$PATH`` when installing this
```

Comment:
CMake isn't strictly required according to .install.sh line 16, but system CMake is better.